### ISSUE TITLE
can't find the download button for downloading files for yaapu x20 ethos

### ISSUE THREAD

**Issue by williajgr** [FrSkyRC/ETHOS-Feedback-Community#2074]
can't find the download button for downloading files for yaapu x20 ethos.  i feel really stupid.  looked all day.

**Comment by mawzthefinn** [comment]
You'll want to go to the Yaapu Github for that script

https://github.com/yaapu/FrskyTelemetryScript/tree/dev/ETHOS

**Comment by williajgr** [comment]
i have looked there all day yesterday and can't find the download button.

**Comment by mawzthefinn** [comment]
That's more of an issue to bring up on Yaapu's Github, which does not belong to FrSky.

That said, if you go up one level to:

https://github.com/yaapu/FrskyTelemetryScript/tree/dev

And click on the Green 'Code' button, it will let you download a Zip file of all the versions, including the ETHOS version.

**Comment by williajgr** [comment]
Thanks, when I got to that page I clicked on ethos the page you sent me to originally and saw the files I wanted but no way to download them.  I wouldn’t have thought I had to download everything.  Didn’t mean to put in on FrSky get hub.  Thought I was in yaapu.   Thanks again for the help.    JoeSent from my iPhoneOn Nov 12, 2022, at 12:04 PM, mawzthefinn ***@***.***> wrote:﻿
That's more of an issue to bring up on Yaapu's Github, which does not belong to FrSky.
That said, if you go up one level to:
https://github.com/yaapu/FrskyTelemetryScript/tree/dev
And click on the Green 'Code' button, it will let you download a Zip file of all the versions, including the ETHOS version.
